packages/cli/yarn.lock

```diff
@@ -0,0 +1,6884 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
yarn.lock files should not be generated / committed within package directories. Please read up on how "lerna + yarn monorepos" work here: https://leerob.io/blog/monorepo-lerna-yarn-workspaces

The steps to install dependencies, build packages, and run tests can be found in the root README of the monorepo. Additionally after you install deps + build packages, you can run tests within a single package by opening a terminal there and running `yarn test` or `yarn test:ci`. Feel free to send messages in the discord if you'd like some help getting adjusted to this setup.